front-end/src/IndividualShow.js

```diff
@@ -0,0 +1,188 @@
+    progress.push(season);
+    progress.push(episode);
+  });
+  return (
+    <form onSubmit={(e) => saveProgressData()} >
+      <label for="season">Current Season: </label>
+      <input id="season" class="progress" value={season} ref={refSeason} />
+      <br /> <br />
+      <label for="episode">Current Episode:</label>
+      <input id="episode" class="progress" value={episode} ref={refEpisode} />
+      <br /><br />
+      <input type="submit" value="Save Progress" />
+    </form>
+  )
+}
+
+/*the component displays and stores user's chosen platform*/
+
+const PlatformData = () => {
+
+  const refNetflix = useRef();
+  const refPrime = useRef();
+  const refHulu = useRef();
+  const refApple = useRef();
+
+
+  const savePlatform = (() => {
+    //the function will be further developed in sprint 2
+  });
+  return (
+    <div>
+      <p>Select the platform: </p>
+      <form onSubmit={(e) => savePlatform()}>
+
+        <input type="checkbox" id="netflix" value="Netflix" ref={refNetflix} />
+        <label for="netflix">Netflix  </label>
+        <input type="checkbox" id="prime" value="Prime" ref={refPrime} />
+        <label for="prime">Prime  </label>
+        <input type="checkbox" id="hulu" value="Hulu" ref={refHulu} />
+        <label for="hulu">Hulu  </label>
+        <input type="checkbox" id="apple" value="Apple" ref={refApple} />
+        <label for="apple">Apple TV+  </label><br /><br />
+        <input type="submit" value="Save Platform" />
+      </form>
+    </div>
+  )
+}
+
+/*the component displays and stores general info about the show*/
+
+const Description = ({ genre, description, totalEpisodes, isMovieN }) => {
+  const refDescription = useRef();
+  const refIsMovie = useRef();
+  const refGenre = useRef();
+  const refTotalEpisodes = useRef();
+  let movieV = { isMovieN };
+  if (movieV) {
+    movieV = "Yes";
+  } else {
+    movieV = "No";
+  }
+  const saveMovieData = (() => {
+    //the function will be further developed in sprint 2
+  });
+  return (
+    <div class="description">
+      <br /><br />
+      <label for="genre">Genre:  </label>
+      <span id="genre" ref={refGenre}>{genre}</span><br /><br />
+      <label for="description">Description: </label>
+      <textarea id="description" value={description} ref={refDescription} readonly /><br /><br />
+      <label for="totalEpisodes">Total Episodes: </label>
+      <span id="totalEpisodes" ref={refTotalEpisodes}>{totalEpisodes}</span><br /><br />
+      <label for="movie">Is it a movie? </label>
+      <span id="movie" value={movieV} ref={refIsMovie}>{movieV}</span><br /><br />
+    </div>
+  )
+}
+
+/*the component calls other components and displays comprehensive info about the show; 
+values passed as parameters to the main components are generated by the mock data API */
+
+const IndividualShow = (props) => {
+
+  const refTitle = useRef();
+  const refCover = useRef();
+  const returnToShows = (() => {
+    window.location.href = '/my-shows/'
```

Comment:
Please make this go to either the user's ID or to 1 instead. Otherwise it redirects to home. Thanks. Same with all other redirects.

I believe we were planning on having the individual show page being a modal, not its own actual webpage, but we can probably make it work having it be on its own page.